CS342Application/Models/Member.swift

```diff
@@ -11,6 +11,7 @@ enum Member: String, Identifiable, CaseIterable {
     case paulSchmiedmayer = "Paul Schmiedmayer, Ph.D."
     case audreyLin = "Audrey Lin"
     case vishnuRavi = "Vishnu Ravi, MD"
+    case ananyaVasireddy = "Ananya Vasireddy, Student"
```

Comment:
You can remove the "Student" extension to present your name in the same style as the other students.
```suggestion
    case ananyaVasireddy = "Ananya Vasireddy"
```